src/assets/arm_restraints/steel_spheres/steel_spheres.asset.ts

```diff
@@ -18,9 +18,13 @@ DefineAsset({
 		provides: [
 			'Restraint',
 			'Restraint_arms',
+			'Hand_item',
 			'Gloves',
 			'Mittens',
 		],
+		covers: [
+			'Hand_item',
+		],
 	},
```

Comment:
Also, as an idea, since you mentioned wanting to wear leather mittens under the steel mittens: To make that possible in an RP without visual immersion breakage, you could add a "hides: 'Mittens'," here if you want~